sdk/storage/azure-storage-common/src/test/java/com/azure/storage/common/policy/ScrubEtagPolicyTest.java

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.common.policy;
+
+import com.azure.core.http.HttpHeaderName;
+import com.azure.core.http.HttpHeaders;
+import com.azure.core.http.HttpMethod;
+import com.azure.core.http.HttpPipeline;
+import com.azure.core.http.HttpPipelineBuilder;
+import com.azure.core.http.HttpRequest;
+import com.azure.core.http.HttpResponse;
+import com.azure.core.test.SyncAsyncExtension;
+import com.azure.core.test.annotation.SyncAsyncTest;
+import com.azure.core.test.http.MockHttpResponse;
+import com.azure.core.test.http.NoOpHttpClient;
+import com.azure.core.util.Context;
+import reactor.core.publisher.Mono;
+
+import java.net.URL;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ScrubEtagPolicyTest {
+    private static final String ETAG_VALUE = "RU3FchB4PhtZdVy44UQO5CGkyZM";
```

Comment:
Is this a randomly generated string value?

Reply:
yeah, its a dummy value, updated it to a dummy tag.